woocommerce/woocommerce
# Package Release Workflow: Rename branch to avoid branch protection rules

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Allow multiple runs of Package Release workflow in a given day without overwriting an existing branch. This PR adds the hours and minutes to the branch name to avoid conflict.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. See test PR https://github.com/woocommerce/woocommerce/pull/41733. This should be sufficient. Ensure the branch name has hours and minutes so its unique enough to not get overwritten.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
